examples/add_points_with_features.py

```diff
@@ -46,13 +47,14 @@
 def toggle_point_annotation(viewer):
     selected_points = list(points_layer.selected_data)
     if len(selected_points) > 0:
-        good_point = points_layer.properties['good_point']
+        good_point = points_layer.features['good_point']
         good_point[list(selected_points)] = ~good_point[list(selected_points)]
```

Comment:
```suggestion
        good_point[selected_points] = ~good_point[selected_points]
```

Not relevant these changes, so feel to ignore, but `selected_points` is already a `list` here.